packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -295,6 +295,22 @@ describe('<Tabs />', () => {
       expect(handleChange.args[0][1]).to.equal(1);
     });
 
+    it('should not call onChange when already selected', () => {
+      const handleChange = spy();
+      const { getAllByRole } = render(
+        <Tabs value={0} onChange={handleChange}>
+          <Tab />
+          <Tab />
+        </Tabs>,
+      );
+
+      fireEvent.click(getAllByRole('tab')[1]);
+      expect(handleChange.callCount).to.equal(1);
```

Comment:
That hole block is already tested above

Reply:
What Im trying to test is that no matter how many clicks are done to the selected tab, the handler won't trigger
What should I change the test it to?

Reply:
That block can be removed. We already have a test that asserts that change is called. We only need to add a test that it's not called.